admin/src/pages/auth/auth-page.tsx

```diff
@@ -11,13 +15,74 @@ type AuthPageProps = {
 export function AuthPage({ authorizationEndpoint }: AuthPageProps) {
 	return (
 		<Page>
-			<Button
-				appearance="primary"
-				onClick={() => openInBrowser(authorizationEndpoint)}
-				iconAfter={<ArrowRightIcon label="" size="medium" />}
+			<div
+				css={css`
+					align-items: center;
+					display: flex;
+					flex-direction: column;
+					gap: 24px;
+				`}
 			>
-				Continue
-			</Button>
+				<ConnectBanner />
+				<div
+					css={css`
+						font-weight: 500;
+						font-size: 24px;
+						color: ${token('color.text')};
+					`}
+				>
+					Connect Figma to Jira
```

Comment:
This might be tricky to get a quick solve for, but have we thought about / are we concerned about i18n for non-English speakers?

Reply:
agreed that we should definitely look into this! IMO this is worthy of its own project (sans something that exists and would easily plug and play). If you know of any i18n/translation libraries, I'm all ears!